pkg/sink/sink.go

```diff
@@ -134,6 +135,10 @@ func createResource(rt json.RawMessage, restClient restclient.Interface, discove
 	if err != nil {
 		log.Print(err)
 	}
+	rt, err = sjson.SetBytes(rt, "metadata.labels."+triggersv1.LabelEscape+triggersv1.EventIDLabelKey, eventId)
+	if err != nil {
+		log.Print(err)
```

Comment:
Should we return here instead of logging and continuing on?

Reply:
I think returning probably makes most sense too. If we expect the label and it isn't there we won't ever know, but we could alternatively not create resources, which seems safer.

Reply:
I'm not sure.   This is same way as the event listener name label.  I leave it for now.

Reply:
OK  I add return.